portfolio/src/main/webapp/portfolio.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
It looks like you added portfolio.html again in another commit. I checked the files and there aren't two portfolio.html files, but it could be helpful for you to double check.

Reply:
Did it get added in a commit removed in another and then added again? Regardless I don't think this will be an issue.

Reply:
I don't see any duplicate files.

Reply:
Yeah I agree that it is not an issue. Just want to double check if I am missing something.